src/TS/services/runInfoDomManipulation.ts

```diff
@@ -104,12 +113,27 @@ export function getCheckedLaps() {
   return lapArray;
 }
 
+export function clearCheckBoxes() {
+  const lapList = document.querySelectorAll('input');
+
+  if (!lapList) {
+    throw new Error('Seems like an error from our side!');
+  }
+
+  for (const lap of lapList) {
+    if (lap.checked) {
+      lap.checked = false;
+    }
+  }
+}
+
 export function startSpinner() {
   const loader: HTMLElement | null = document.querySelector('.loader');
   if (!loader) {
     throw new Error('Seems like an error from our side!');
   }
   loader.style.display = 'block';
+  loader.style.zIndex = '1';
 
   const page: HTMLElement | null = document.querySelector(
```

Comment:
I see that there are times when you specify the time of the element you're getting from querySelector and times when you don't. I think you can leave it out and let typescript infer the type

Reply:
The type that `TS` infers (`Element`) does not have a style object. However the type that is specified here `HTMLElement` does.